integration-libs/opf/cta/components/opf-cta-scripts/opf-cta-scripts.component.html

```diff
@@ -1,13 +1,8 @@
-<ng-container *ngIf="ctaHtmls$ | async as ctaHtmls; else loading">
+<ng-container *ngIf="ctaHtmls$ | async as ctaHtmls">
   <ng-container *ngIf="ctaHtmls?.length">
     <cx-opf-cta-element
       *ngFor="let ctaHtml of ctaHtmls"
       [ctaScriptHtml]="ctaHtml"
     ></cx-opf-cta-element>
   </ng-container>
 </ng-container>
-<ng-template #loading>
```

Comment:
Removed spinner as it is displayed even when cta returns empty array.
We want cta to be invisible when it is not used.